nixos/doc/manual/configuration/wayland.chapter.md

```diff
@@ -28,4 +28,4 @@ able to share your screen, you might want to activate this option:
 
 and configure Pipewire using
 [](#opt-services.pipewire.enable)
-and related options.
+and related options (see the [wiki page about Sway](https://wiki.nixos.org/wiki/Sway)).
```

Comment:
This doesn't make sense to put here, the sway wiki isn't really relevant to pipewire.

It'd make more sense as an additional paragraph.

Reply:
Done in [0c18dfa](https://github.com/NixOS/nixpkgs/pull/349344/commits/0c18dfa1a593557a49319e66501fde986ec8a4c8).